hs_core/models.py

```diff
@@ -3784,11 +3784,6 @@ def _update_relation_meta(relation_meta_obj):
                     relation_updated = True
             return relation_updated
 
-        version_relation = self.metadata.relations.all().filter(type='isVersionBy').first()
-        version_relation_updated = False
```

Comment:
@pkdash Noticed the wrong type name ```isVersionBy``` in the removed code, which was missed in the initial code review and fixed implicitly as a result of this.

Reply:
@hyi I also noticed that when making this last code change. I will be doing some cleanup work related to Relation class after this is deployed to production. Part of that cleanup work is to avoid using strings for relation type - instead use an enum type.

Reply:
@pkdash using enum type for relation type sounds good which will avoid this type of string typo error.